README.md

```diff
@@ -146,25 +145,54 @@ def deps do
 end
 ```
 
-Also, please don't forget to copy or merge the config from evision to the top-level project.
+### Configuration
 
-If you don't have any other config in the top-level project, then you can just copy the whole `config` directory
+`evision` will compile a subset of OpenCV functionality. You can configure the enabled modules in your `config` files:
 
-```shell
-mix deps.get
-cp -a deps/evision/config config
+```elixir
+config :evision, enabled_modules: [
+  :calib3d,
+  :core,
+  :features2d,
+  :flann,
+  :highgui,
+  :imgcodecs,
+  :imgproc,
+  :ml,
+  :photo,
+  :stitching,
+  :ts,
+  :video,
+  :videoio,
+  :dnn
+]
 ```
 
-Otherwise, you can import this config file at the end of existing `config/config.exs`
+If a module is not specified in `:enabled_modules`, it may still be compiled if all requirements are present in your machine.
+You can enforce only the `:enabled_modules` to be compiled by changing the compilation mode:
```

Comment:
Please let me know if this paragraph is correct and if I understood how `:compile_mode` works properly.

Reply:
Oh, for this part, except for the `except_disabled_modules` mode, `: enabled_modules` is like what you want to enable, but the actual availability will depend on some other factors, for example

- if module `A` depends on module `B`, but `B` is in the list of `:disabled_modules`, then `A` will be disabled due to the dependency
- or, if module `A` depends on one or more third-party libraries, but these libraries are not available on your machine, then `A` will be disabled by CMake.

Reply:
In `except_disabled_modules` mode, then the disabled modules, say set `D`, and modules depend on any module in set `D` will also be disabled.